### PR TITLE
[script.tag-generator] 0.5.1

### PR DESCRIPTION
**Pull request by Razzeee**
#478

### CODE REVIEW

File: script.tag-generator/addon.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.tag-generator"
+       name="Tag Generator"
+       version="0.5.1"
```
Comment (Razzeee) [inline script.tag-generator/addon.xml]: @wellspokenman please be aware that this changes your versioning system in a "big" way. Let me know it it's okay. You might also want to do this to your github repo then.

File: script.tag-generator/main.py [hunk 2]
```diff
@@ -0,0 +1,521 @@
...
+try:
+    import simplejson
+    import trakt
+    from trakt import users
+except:
+    if os.name == "nt":
+        slash = "\\"
+    else:
+        slash = "/"
+    sys.path.append(os.path.abspath(os.path.dirname(__file__)+slash+"resources"+slash+"lib"))
```
Comment (enen92) [inline script.tag-generator/main.py]: I doubt this is needed

**Comment by Razzeee** [comment]
@wellspokenman can you submit a fanart?

File: script.tag-generator/main.py (hunk 2 above)
Comment (romanvm) [inline script.tag-generator/main.py]: `os.path.join()` is the correct method of platform-independent joining path strings in Python

**Comment by wellspokenman** [comment]
@Razzeee done, and resized icon.png: https://github.com/wellspokenman/tag-generator/commit/d82f3ae4597ee79202f2a3a7cea57c960d53226d

**Comment by enen92** [comment]
@Razzeee feel free to merge

**Pull request merged**